### ISSUE TITLE
Decide on whether or not to continue built-in Captcha support

### ISSUE THREAD

**Issue by rvanbaalen** [validformbuilder/validformbuilder#7]
There have been some rumours around that Captcha fields arent't working the way they're supposed to. 
- [x] Decide on the future of Captcha in ValidForm Builder
- [x] Validate Captcha fields are still working server-side (add captcha, parse field) **not working**
- [x] Validate Captcha fields are still working client-side (re-generate Captcha, etc.) **not working**

@treffynnon, feel free to throw in your two cents on the subject. Should ValidForm Builder contain captcha functionality out of the box or should VFB users add third party Captcha libraries themselves?

TL;DR:
**Don't touch the Captcha element**
